### PR TITLE
4346 Add Exit and Back buttons on Status Checker

### PR DESCRIPTION
**Pull request by Fbasham**
### Description
per title

### Related Azure Boards Work Items
[AB#4346](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/4346)

### Screenshots (if applicable)
image

image

### CODE REVIEW

File: frontend/app/routes/$lang/_public/status/myself.tsx [hunk 1]
```diff
@@ -181,6 +183,9 @@ export default function StatusCheckerMyself() {
           <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12" endIcon={faChevronRight}>
             {t('status:myself.check-another')}
           </ButtonLink>
+          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block font-lato font-semibold">
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang/_public/status/myself.tsx:186]: The link's URL should be `https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#status` in English or `https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#etat` in French

File: frontend/app/routes/$lang/_public/status/child.tsx [hunk 2]
```diff
@@ -294,6 +296,9 @@ export default function StatusCheckerChild() {
           <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12" endIcon={faChevronRight}>
             {t('status:child.check-another')}
           </ButtonLink>
+          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block font-lato font-semibold">
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang/_public/status/child.tsx:299]: The link's URL should be `https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#status` in English or `https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#etat` in French

File: frontend/app/routes/$lang/_public/status/child.tsx [hunk 3]
```diff
@@ -294,6 +296,9 @@ export default function StatusCheckerChild() {
...
+            {t('status:myself.exit-btn')}
```
Comment (nicholas-ly) [inline frontend/app/routes/$lang/_public/status/child.tsx:300]:
```suggestion
            {t('status:child.exit-btn')}
```

File: frontend/app/routes/$lang/_public/status/child.tsx [hunk 4]
```diff
@@ -368,9 +373,15 @@ export default function StatusCheckerChild() {
                 </>
               )}
             </div>
-            <LoadingButton variant="primary" id="submit" loading={isSubmitting} data-gc-analytics-formsubmit="submit" endIcon={faChevronRight}>
-              {t('status:child.form.submit')}
-            </LoadingButton>
+            <div className="flex flex-wrap items-center gap-3">
+              <ButtonLink id="back-button" routeId="$lang/_public/status/index" params={params} disabled={isSubmitting}>
+                <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
+                {t('status:myself.form.back-btn')}
```
Comment (nicholas-ly) [inline frontend/app/routes/$lang/_public/status/child.tsx:379]:
```suggestion
                {t('status:child.form.back-btn')}
```

File: frontend/app/routes/$lang/_public/status/child.tsx (hunk 2 above)
Comment (nicholas-ly) [inline frontend/app/routes/$lang/_public/status/child.tsx:299]: I feel this link should have the same font style as the rest of the links in the application.

```suggestion
          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block">
```

File: frontend/app/routes/$lang/_public/status/myself.tsx (hunk 1 above)
Comment (nicholas-ly) [inline frontend/app/routes/$lang/_public/status/myself.tsx:186]: I feel this link should have the same font style as the rest of the links in the application.

```suggestion
          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block">
```

File: frontend/public/locales/en/status.json [hunk 5]
```diff
@@ -66,6 +66,8 @@
     "page-title": "Status Checker",
     "status-heading": "Status",
     "check-another": "Check another status",
+    "exit-btn": "Exit status checker",
+    "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan.html",
```
Comment (sebastien-comeau) [inline frontend/public/locales/en/status.json:70]: Based on Figma, it must be:

```suggestion
    "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#status",
```

File: frontend/public/locales/en/status.json [hunk 6]
```diff
@@ -90,6 +93,8 @@
...
```
Comment (sebastien-comeau) [inline frontend/public/locales/en/status.json:97]: Based on Figma, it must be:

```suggestion
    "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#status",
```

File: frontend/public/locales/fr/status.json [hunk 7]
```diff
@@ -66,6 +66,8 @@
     "page-title": "Vérificateur de l'état",
     "status-heading": "État",
     "check-another": "Vérifier un autre état",
+    "exit-btn": "Quitter le vérificateur de l'état",
+    "exit-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires.html",
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/status.json:70]: Based on Figma

```suggestion
    "exit-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#etat",
```

File: frontend/public/locales/fr/status.json [hunk 8]
```diff
@@ -90,6 +93,8 @@
...
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/status.json:97]: Based on Figma
```suggestion
    "exit-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#etat",
```

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**